web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/var/isFunction.js

```diff
@@ -0,0 +1,13 @@
+define( function() {
+	"use strict";
+
+	return function isFunction( obj ) {
+
+      // Support: Chrome <=57, Firefox <=52
```

Comment:
:no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).